examples/linear_model/plot_sparse_logistic_regression_20newsgroups.py

```diff
@@ -18,6 +18,7 @@
 input features would be to use univariate feature selection followed by a
 traditional (l2-penalised) logistic regression model.
 """
+from __future__ import division
```

Comment:
Yeah, true, this seems to be another way to approach this. @jnothman your opinions of using this instead?